### PR TITLE
Artifactory license

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Add support for providing the artifactory license key as a parameter to helm install/upgrade instead of just using a secret name

### CODE REVIEW

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -235,6 +235,7 @@ artifactory:
   ## ARTIFACTORY_HOME/etc/artifactory.cluster.license and loaded at run time.
   ## The dataKey should be the name of the secret data key created.
   license:
+    licenseKey:
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml]: Add a note that only `licenseKey` or `secret` should be used or that `secret` will override `licenseKey` if both are passed.

File: stable/artifactory/values.yaml [hunk 2]
```diff
@@ -175,6 +179,7 @@ artifactory:
   ## ARTIFACTORY_HOME/etc/artifactory.lic and loaded at run time.
   ## The dataKey should be the name of the secret data key created.
   license:
+    licenseKey:
```
Comment (eldada) [inline stable/artifactory/values.yaml]: Add a note that only `licenseKey` or `secret` should be used or that `secret` will override `licenseKey` if both are passed.

File: stable/artifactory-ha/values.yaml (hunk 1 above)
Comment (danielezer) [reply stable/artifactory-ha/values.yaml]: @eldada Done

File: stable/artifactory/values.yaml (hunk 2 above)
Comment (danielezer) [reply stable/artifactory/values.yaml]: @eldada Done

**Comment by danielezer** [comment]
@eldada seems like the build is going to timeout for the second time. Is that something we should be worried from?

**Comment by jainishshah17** [comment]
@danielezer Please resolve conflicts.

**Comment by danielezer** [comment]
> @danielezer Please resolve conflicts.

@jainishshah17 Done

**Pull request merged**